tach/cache/package.yml

```
@@ -0,0 +1 @@
+tags: ['tach.cache']
```

This would also affect tach.yml
```suggestion
tags: ['scope:cache']
strict: true
```